src/gdata/apps/service.py

```diff
@@ -25,7 +25,7 @@
       from xml.etree import ElementTree
     except ImportError:
       from elementtree import ElementTree
-import urllib
+import urllib.request, urllib.parse, urllib.error
```

Comment:
where are these being used in this file? Can't see where any lines using them have changed...